src/Countries/Angola.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Angola extends Country
+{
+    public function countryCode(): string
+    {
+        return 'ao';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+
+        return array_merge([
+            'Dia de Ano Novo' => '01-01',
```

Comment:
TIL, Angola speaks Portuguese!

Reply:
Yes, Portuguese is the official language of Angola.